src/Foreign/JavaScript/Types.hs

```diff
@@ -251,10 +252,11 @@ data CallBufferMode
     -- to simplify usage. Users may choose 'BufferRun' instead if they want more control
     -- over flushing the buffer.
     | FlushPeriodically
+      { max_buffer_timeout :: Int
```

Comment:
The standard style here is CamlCase, I would prefer this to be `maxBufferTimeout`.